src/main/java/ch/heigvd/dai/models/BMPHeader.java

```diff
@@ -92,6 +92,22 @@ public void writeHeader(BufferedOutputStream bos) throws IOException {
     bos.write(infoHeader);
   }
 
+  /**
+   * Read the image's height from the header
+   *
+   * @return image height
+   */
+  public int getImageHeight() {
+    if (infoHeader.length < 12) {
+      throw new RuntimeException("[e]: Error, header data is insufficient for reading the height.");
```

Comment:
Don't put "[e] ..." before the error message. This will be added by our error printing mechanism. Instead, write: "Header data is ..."